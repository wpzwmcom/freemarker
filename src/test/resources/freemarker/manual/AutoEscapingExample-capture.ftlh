<#ftl output_format="HTML"><#-- Needed for the Eclipse plugin -->
<#assign captured><b>Test</b></#assign>
Just a string: ${"<b>Test</b>"}
Captured output: ${captured}