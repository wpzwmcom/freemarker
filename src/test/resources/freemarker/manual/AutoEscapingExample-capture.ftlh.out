Just a string: &lt;b&gt;Test&lt;/b&gt;
Captured output: <b>Test</b>